007WS-cry/readme.md

```
@@ -0,0 +1,303 @@
+
+POST /auth/register/{username}{password}
+
+### 请求参数
+
+|名称|位置|类型|必选|说明|
+|---|---|---|---|---|
+|username|path|string| 是 |none|
+|password|path|string| 是 |none|
+
+> 返回示例
+
```

Markdownlint

Blank line inside blockquote

(MD028, no-blanks-blockquote)